zarr/v3/config.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from asyncio import AbstractEventLoop
+from typing import Literal, Optional
+from attr import frozen
+
+
+@frozen
+class SyncConfiguration:
+    concurrency: Optional[int] = None
+    asyncio_loop: Optional[AbstractEventLoop] = None
```

Comment:
this was a half hearted attempt at rethinking the config topic. I expect this to go away soon.